demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -203,20 +203,16 @@ def _gather_facts() -> Dict[str, Any]:
             )
             logger.warning(f"can't locate content repo {e}")
         # Get global requirements file
-        pipfile_dir = Path(__file__).parent / "resources"
+        test_requirements_dir = Path(__file__).parent / "resources"
         try:
-            pipfile_lock_path = pipfile_dir / "pipfile_python3/Pipfile.lock"
-            with open(file=pipfile_lock_path) as f:
-                lock_file: dict = json.load(fp=f)["develop"]
-                facts["requirements_3"] = [
-                    key + value["version"]
-                    for key, value in lock_file.items()  # type: ignore
-                ]
-                logger.debug(
-                    "Test requirements successfully collected for python 3:\n"
-                    f" {facts['requirements_3']}"
-                )
-            python2_requirements = pipfile_dir / "pipfile_python2/dev-requirements.txt"
+            python3_requirements = (
+                test_requirements_dir / "python3_requirements" / "dev-requirements.txt"
+            )
+            facts["requirements_3"] = python3_requirements.read_text().strip().split("\n")  # type: ignore
+
+            python2_requirements = (
+                test_requirements_dir / "python2_requirements" / "dev-requirements.txt"
+            )
```

Comment:
can we reuse the ones from above?

Reply:
we don't need it anymore, I'll delete it